package.json

```diff
@@ -21,13 +21,13 @@
     "deploy": "yarn clean && yarn build-bundle && git add dist/ && git commit -m \"Build + package for release.\" && yarn version --patch && git push origin --tags HEAD"
   },
   "engines": {
-    "node": "^14.19.1"
+    "node": "^14"
   },
   "dependencies": {
     "@avcs/autosuggest": "^1.7.5",
     "@floip/expression-evaluator": "^2.1.0",
     "@floip/expression-parser": "^1.11.3",
-    "@floip/flow-runner": "^1.0.0-rc4",
+    "@floip/flow-runner": "https://github.com/FLOIP/flow-runner.git#3a3020c3b14a27d14b62003c3fc54270cf287ad9",
```

Comment:
see https://github.com/FLOIP/flow-runner/tree/feature/VMO-5662/use-base-store-in-blocks

Reply:
@bulatgab 
- [ ] Once we bumped flow-runner version & we published it, let's re-use the semantic version here.